plugins/out_stackdriver/stackdriver.c

```diff
@@ -1146,15 +1268,29 @@ static int stackdriver_format(struct flb_config *config,
             entry_size += 1;
         }
 
+        /* Extract insertId */
+        in_status = validate_insert_id(&insert_id_obj, obj);
```

Comment:
Change the function name to `extract_insert_id` ?

Reply:
`in_status` means the status of insertId - invalid, not present or valid.
There has been a parameter named `insert_id_extracted`, which presents whether insertId has been extracted.
What do you think?

Reply:
I can change `in_status` to `insert_id_status` to make it clear